bedrock/foundation/templates/foundation/annualreport/2019/index.html

```diff
@@ -69,7 +69,7 @@ <h2><span class="highlight dark">Our impact</span></h2>
       products; and choosing technology made by companies who share your
       vision for a healthier internet. We invite anyone interested in
       contributing to building a better internet to
-      <a href="{{ url('firefox.new') }}">download Firefox</a>, to
+      <a href="https://www.firefox.com/?redirect_source=mozilla-org">download Firefox</a>, to
```

Comment:
I'm wondering … is this a redirect? I'd say that should be reserved for folks sent from fx/new to fxc. But this will most likely stay, as a normal full-blown link to another moco site. So either no params, or some utm content to track where the click originated from?

Reply:
(meant for all the occurrences…)